docs/log-in-as-po.md

```diff
@@ -1,24 +1,55 @@
 # Log in as public officer
 
-1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in with TechPass**.
-<kbd>![log-in-with-techpass](assets/images/access-sgts-services-using-techpass/first.png)</kbd>
-2. Sign in to your TechPass account or choose your TechPass account.
-<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+Public officer can log in to TechPass using your non-SE GSIB device or your Government Managed Device (GMD).
```

Comment:
@kpslp to make it clear, can we say  PO can log in with TechPass ...as log in to TechPass may also mean TechPass portal and currently cant do that using GMD.

Reply:
instead of 
Public officer - change to - Public officers

 "your " to "their"